iwyu_output.cc

```diff
@@ -475,7 +477,6 @@ string OneIncludeOrForwardDeclareLine::LineNumberString() const {
   return buf;
 }
 
-
```

Comment:
I tend to overthink empty lines. Why was it there before? Why was it removed? In this case extra empty line might have been used to separate `OneIncludeOrForwardDeclareLine` methods from `IwyuFileInfo` methods.